### PR TITLE
Changelog for 3.2

### PR DESCRIPTION
**Pull request by tsiegleauq**
WIP

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -8,6 +8,50 @@ Version 3.2 (unreleased)
 ========================
 `Milestone <https://github.com/OpenSlides/OpenSlides/milestones/3.2>`_
 
+========================
+"is:pr is:closed milestone:3.2 sort:updated-desc"
+last entry: #5323
+========================
```
Comment (tsiegleauq) [inline CHANGELOG.rst:14]: Helps to keep the changelog up to date. Delete later.

**Comment by emanuelschuetze** [comment]
@tsiegleauq can you update the changelog please?

**Comment by tsiegleauq** [comment]
updated

**Comment by tsiegleauq** [comment]
updated

**Comment by tsiegleauq** [comment]
sorting to create changelogs:

"is:pr milestone:3.2 sort:created-desc"
last entry: #5429

**Comment by tsiegleauq** [comment]
updated

**Pull request merged**